### DISCUSSION TITLE
Checkbox

### DISCUSSION THREAD

**Discussion by ksamirdev** [mdx-editor/editor#292] [Q&A]
How do I disable the checkbox?

**Comment by noteverseio** [comment]
You can write your own list plugin  and remove the check list in your custom list plugin.

A better approach can be MDXEditor to expose configuration flag to control what list type to be enabled.

**Reply by ksamirdev** [reply]
I don't find any documentation for configuration flag. Can you provide example please?

**Reply by noteverseio** [reply]
The configuration flag is yet to be implemented;)

**Reply by ksamirdev** [reply]
Any other way to achieve that right now?

**Reply by murtazablt** [reply]
You can pass options  prop to ListToggle component so: `<ListsToggle options={['bullet', 'number']} />`
